uploadNeuroDB/NeuroDB/MRI.pm

```diff
@@ -234,7 +234,7 @@ sub getSessionID {
 	# fixme ask Jon ... is this still useful?
     # determine the centerID and new visit number (which is now deprecated) if getPSC() failed.
 	if($centerID == 0) {
-            $query = "SELECT IFNULL(MAX(VisitNo), 0)+1 AS newVisitNo, CenterID FROM session WHERE CandID=".$dbh->quote($subjectIDref->{'CandID'})." GROUP BY CandID";
+            $query = "SELECT IFNULL(MAX(VisitNo), 0)+1 AS newVisitNo, CenterID FROM session WHERE CandID=".$dbh->quote($subjectIDref->{'CandID'})." GROUP BY CandID newVisitNo, CenterID";
```

Comment:
Thanks! I fixed it. I'm not sure actually, I was setting up MRI on my sandbox in combination with what PreventAD has currently. Maybe Cecile would understand better why I was hitting this condition. If I remember correctly yes she did mention something about a new visit being created...and I do believe our scripts create new sessions from the back end, not sure if that's normal or not.